js/src/tooltip.js

```diff
@@ -486,6 +487,17 @@ class Tooltip {
   }
 
   _addAttachmentClass(attachment) {
+    switch (attachment) {
+      case 'right':
+        attachment = 'end'
+        break
+      case 'left':
+        attachment = 'start'
+        break
+      default:
+        break
+    }
```

Comment:
it seems it's the same logic in our Popover so can add a protected method to do that which will be used in the Popover and Tooltip ?

BTW I'm not a big fan of `switch`s for two cases, I would prefer something like that:
```js
updateAttachment(attachment) {
  if (attachment === 'right') {
    return 'end'
  }

  if (attachment === 'left') {
    return 'start'
  }

  return attachment
}

Reply:
Indeed, much simpler <3